packages/code/src/Code.tsx

```diff
@@ -80,6 +92,131 @@ function getWrapperVariantStyle(variant: Variant): string {
   `;
 }
 
+function getSidebarVariantStyle(variant: Variant): string {
+  const colors = variantColors[variant];
+
+  switch (variant) {
+    case 'light':
+      return css`
+        border-color: ${colors[1]};
+        background-color: white;
+      `;
+    case 'dark':
+      return css`
+        border-color: ${colors[1]};
+        background-color: ${colors[1]};
+      `;
+  }
+}
+
+function getCopyButtonStyle(variant: Variant, copied: boolean): string {
+  if (copied) {
+    return css`
+      align-self: center;
+      color: ${uiColors.white};
+      background-color: ${uiColors.green.base};
+    `;
+  }
+
+  if (variant === Variant.Dark) {
+    return css`
+      align-self: center;
+      color: ${uiColors.gray.base};
+      background-color: ${uiColors.gray.dark3};
+    `;
+  }
+
+  return css`
+    align-self: center;
+    color: ${uiColors.gray.base};
+  `;
+}
+
+const ScrollState = {
+  None: 'none',
+  Left: 'left',
+  Right: 'right',
+  Both: 'both',
+};
```

Comment:
this can probably be `as const` right?

Reply:
And then we can turn this into a type as well so that we can use it in the getScrollShadowStyle function